From ltsi-dev-bounces@lists.linuxfoundation.org Mon May 28 23:23:26 2012
From: "Shimoda, Yoshihiro" <yoshihiro.shimoda.uh@renesas.com>
Date: Tue, 29 May 2012 15:22:23 +0900
Subject: [LTSI-dev] [PATCH 01/12] usb: gadget: add platform module alias where it is missing
To: ltsi-dev@lists.linuxfoundation.org
Message-ID: <4FC46B1F.6040202@renesas.com>


>From 2db193dbaf8523e7ebfd9d16add47a85435b6b54 Mon Sep 17 00:00:00 2001
From: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Date: Wed, 29 Jun 2011 16:41:55 +0300
Subject: usb: gadget: add platform module alias where it is missing

Without it udev won't be able to load the driver once it notices the
device unbound.

Signed-off-by: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Signed-off-by: Felipe Balbi <balbi@ti.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
(cherry picked from commit 86081d7be34e0f80dc5c46391ec927fc4f350462)

Conflicts:

	drivers/usb/gadget/ci13xxx_msm.c
	drivers/usb/gadget/mv_udc_core.c
	drivers/usb/gadget/net2272.c
	drivers/usb/gadget/s3c-hsudc.c

Signed-off-by: Yoshihiro Shimoda <yoshihiro.shimoda.uh@renesas.com>
---
 drivers/usb/gadget/r8a66597-udc.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/usb/gadget/r8a66597-udc.c b/drivers/usb/gadget/r8a66597-udc.c
index 6dcc1f6..600a2e7 100644
--- a/drivers/usb/gadget/r8a66597-udc.c
+++ b/drivers/usb/gadget/r8a66597-udc.c
@@ -1679,6 +1679,7 @@ static struct platform_driver r8a66597_driver = {
 		.name =	(char *) udc_name,
 	},
 };
+MODULE_ALIAS("platform:r8a66597_udc");

 static int __init r8a66597_udc_init(void)
 {
-- 
1.7.1
_______________________________________________
LTSI-dev mailing list
LTSI-dev@lists.linuxfoundation.org
https://lists.linuxfoundation.org/mailman/listinfo/ltsi-dev

